backend/src/main/java/fr/zelytra/session/player/Player.java

```diff
@@ -73,6 +76,14 @@ public void setSessionId(String sessionId) {
         this.sessionId = sessionId;
     }
 
+    public String getClientVersion() {
+        return clientVersion;
+    }
+
+    public void setClientVersion(String clientVersion) {
+        this.clientVersion = clientVersion;
+    }
```

Comment:
why is that a thing ? The player object shouldn't be built at all server side when the version doesnt match.

Reply:
Those are basic java setter and getter of the class. I don't understand your misconcetion ?